src/ModelManager.ts

```diff
@@ -17,6 +17,7 @@ import { Model } from './Model';
 import { ModelClient } from './ModelClient';
 import { ModelStore } from './ModelStore';
 import { PathUtils } from './PathUtils';
+import {Utils} from "./Utils";
```

Comment:
`import { Utils } from "./Utils";` to make linter happy

Reply:
Done.